plasmapy/particles/__init__.py

```diff
@@ -46,3 +57,30 @@
 deuteron = Particle("D 1+")
 triton = Particle("T 1+")
 alpha = Particle("He-4 2+")
+
+
+def json_load_particle(json_file_object):
+    """
+    Returns the appropriate object from input JSON file object
+
+    Parameters
+    ----------
+    json_file_object: file object
+        file object of the JSON file that contains the JSON string describing
+        the particle
+    """
+    json_string = None
+    json_string = json_file_object.read()
+    return json_loads_particle(json_string)
+
+
+def json_loads_particle(json_string):
+    """
+    Returns the appropriate object from input JSON string representation
+
+    Parameters
+    ----------
+    json_string: string
+        JSON string that represents a particle
+    """
+    return json.loads(json_string, cls=ParticleJSONDecoder)
```

Comment:
I think this deserves a separate file - it's a good practice to keep code in `__init__.py` to a minimum.

Reply:
Thank you! That is a good idea. Any hints on a suitable file name? I was thinking something like `particle_json_utils.py`

Reply:
`json.py` or `serialization.py` are the first two ideas that come to mind. I think I would lean towards the later.

Reply:
Thank you! Will change the name to `serialization.py`